### PR TITLE
Adding documentation for new APIs in DecompressionMethods and MultipartContent classes.

### PR DESCRIPTION
**Pull request by jozkee**
Documented APIs:
* System.Net.DecompressionMethods
  * Brotli
  * All

* System.Net.Http.MultipartContent
  * CreateContentReadStreamAsync - this one probably has serious problems in the first paragraph of the Remarks section since I was struggling too hard to find the proper words to describe it.

### CODE REVIEW

**Comment by Jozkee** [comment]
@dotnet/ncl @stephentoub 
@mairaw @rpetrusha @carlossanlop

File: xml/System.Net.Http/MultipartContent.xml [hunk 1]
```diff
@@ -197,9 +197,20 @@
       </ReturnValue>
       <Parameters />
       <Docs>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
-        <remarks>To be added.</remarks>
+        <summary>Serialize the HTTP content to a stream using the multipart/* encoding as an asynchronous operation.</summary>
```
Comment (rpetrusha) [inline xml/System.Net.Http/MultipartContent.xml]:
```suggestion
        <summary>Serializes the HTTP content to a stream using the multipart/* encoding as an asynchronous operation.</summary>
```

File: xml/System.Net.Http/MultipartContent.xml [hunk 2]
```diff
@@ -197,9 +197,20 @@
...
+        <returns>The task object representing the asynchronous operation.</returns>
+        <remarks>
+          <format type="text/markdown"><![CDATA[  
+  
+## Remarks  
+ This method overrides <xref:System.Net.Http.HttpContent.CreateContentReadStreamAsync%2A?displayProperty=nameWithType> to use a custom stream that contains an array with each HTTP content and its boundary encoded and serialized to a <xref:System.IO.MemoryStream> instance.
```
Comment (rpetrusha) [inline xml/System.Net.Http/MultipartContent.xml]:
```suggestion
 This method overrides <xref:System.Net.Http.HttpContent.CreateContentReadStreamAsync%2A?displayProperty=nameWithType> to use a custom stream that contains an array, with each HTTP content and its boundary encoded and serialized to a <xref:System.IO.MemoryStream> instance.
```

File: xml/System.Net.Http/MultipartContent.xml [hunk 3]
```diff
@@ -197,9 +197,20 @@
...
+
+ This operation will not block. The returned <xref:System.Threading.Tasks.Task%601> object will complete after all of the content has been written to the memory stream.  
+  
+ Once the operation completes, the <xref:System.Threading.Tasks.Task%601.Result%2A> property on the returned task object contains the stream that represents the multipart/* encoded HTTP content. The returned stream can then be used to read the content using various stream APIs.
```
Comment (rpetrusha) [inline xml/System.Net.Http/MultipartContent.xml]:
```suggestion
 Once the operation completes, the <xref:System.Threading.Tasks.Task%601.Result%2A?displayProperty=nameWithType> property on the returned task object contains the stream that represents the multipart/* encoded HTTP content. The returned stream can then be used to read the content using various stream APIs.  
```

File: xml/System.Net/DecompressionMethods.xml [hunk 4]
```diff
@@ -85,7 +85,7 @@
       </ReturnValue>
       <MemberValue>4</MemberValue>
       <Docs>
-        <summary>To be added.</summary>
+        <summary>Use Brotli compression-decompression algorithm.</summary>
```
Comment (rpetrusha) [inline xml/System.Net/DecompressionMethods.xml]:
```suggestion
        <summary>Use the Brotli compression-decompression algorithm.</summary>
```

**Comment by rpetrusha** [comment]
Thanks for making the additional changes, @Jozkee. I'll merge your PR now.

**Pull request merged**